DQMOffline/Trigger/src/HLTMuonMatchAndPlot.cc

```diff
@@ -374,37 +376,35 @@ vector<size_t> HLTMuonMatchAndPlot::matchByDeltaR(const vector<T1>& collection1,
 
 MuonCollection HLTMuonMatchAndPlot::selectedMuons(const MuonCollection& allMuons,
                                                   const BeamSpot& beamSpot,
-                                                  bool hasRecoCuts,
-                                                  const StringCutObjectSelector<reco::Muon>& selector,
+                                                  double RecoMuonEtaMax,
+                                                  double RecoMuonEtaMin,
+                                                  bool IsMuonGlb,
                                                   double d0Cut,
                                                   double z0Cut) {
```

Comment:
These are all class members: I don't see the reason why to pass them as parameters to the method (it does not seem to me that they get modified by the method)
```suggestion
                                                  ) {
```